### DISCUSSION TITLE
I config tye script, but when i start app, it shows Listening on 127.0.0.1 TCP port 27042

### DISCUSSION THREAD

**Discussion by linsongjie92** [frida/frida#2725] [General]
what i want is that, run my hook.js file, but it stop on listening
This is my config file:
libfrida.config.so
{
  "interaction": {
    "type":"script",
    "path":"/data/local/tmp/hook.js"
  }
}

Please help my solve this problem, thanks.

**Comment by antonshkurenko** [comment]
Did you fix the issue? @linsongjie92

**Comment by antonshkurenko** [comment]
Add `android:extractNativeLibs="true"` to a manifest
